src/components/wallet/UnlockForm.tsx

```diff
@@ -83,6 +88,15 @@ const UnlockForm: React.FC<Props> = (props: Props): JSX.Element => {
     form.resetFields()
   }, [form])
 
+  const onRemove = useCallback(() => {
+    const isConfirmed = window.confirm('remove wallet?')
+    if (isConfirmed) {
+      removeKeystore().then(() => {
```

Comment:
Do we really need `.then` handler here ? `removeKeystore()` changes `KeystoreState` to "locked" state, which should trigger a route change in `WalletView, isn't?

Reply:
yep. u r right